dss-api.yml

```diff
@@ -1375,8 +1375,12 @@ paths:
                 description: Timestamp of bundle creation in RFC3339.
                 type: string
                 format: date-time
+              manifest:
+                type: object
+                description: Manifest data regarding data from the bundle
```

Comment:
Change to "The manifest stored in the DSS. It lists the files in the bundle, and metadata describing those files."

Reply:
Address this comment and I'll approve

Reply:
complete